eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ledger.hs

```diff
@@ -243,6 +256,13 @@ ledgerTransition = do
             )
       else pure certState
 
+  let wdrlAddrs = Map.keysSet . unWithdrawals $ tx ^. bodyTxL . withdrawalsTxBodyL
+  let wdrlCreds = Set.map getRwdCred wdrlAddrs
+  let delegatedAddrs = Map.keysSet . dRepMap . dsUnified $ certDState certState'
+
+  eval (wdrlCreds ⊆ delegatedAddrs)
+    ?! ConwayWdrlNotDelegatedToDRep (wdrlCreds Set.\\ delegatedAddrs)
```

Comment:
If my in github coding is correct this will be much faster, since there is neither Map nor Set conversion unless there is a predicate failure.
```suggestion
  let dUnified = dsUnified $ certDState certState'
      delegatedAddrs = DRepUView dUnified

  all (`UMap.member` delegatedAddrs) wdrlCreds
    ?! ConwayWdrlNotDelegatedToDRep (wdrlCreds Set.\\ Map.keysSet (dRepMap 
 dUnified))
```

This is quite important, cause it is expected we'll have quite a few DReps and doing this conversion for every transaction will be costly.

Reply:
I just realized something very important. We want to disable this functionality for now, until we have the delegation fully implemented in cardano-api. Because otherwise we will not be submit regular transactions with withdrawals, which can be very disruptive for testing.